auth/types/account_retriever.go

```diff
@@ -2,18 +2,28 @@ package types
 
 import (
 	"fmt"
+	cliContext "github.com/cosmos/cosmos-sdk/client/context"
 
 	"github.com/maticnetwork/heimdall/types"
 )
 
+var (
+	NQuerier NodeQuerier
```

Comment:
can we do something to not to have global vatiable?

Reply:
I am afraid we can't, as we want this variable to be a mockNodeQuerier in tests and a pure &cliContext.CLIContext{} with normal execution